FFmpegInterop/FFmpegInteropMSS.cpp

```diff
@@ -296,6 +297,31 @@ MediaPlaybackItem^ FFmpegInteropMSS::CreateMediaPlaybackItem(TimeSpan startTime,
 	}
 }
 
+IAsyncAction^ FFmpegInterop::FFmpegInteropMSS::ParseExternalSubtitleStream(IRandomAccessStream ^ stream)
```

Comment:
Can we rename this to something like AddExternalSubtitleAsync()?

Reply:
Agreed. I will do it in the next commit (probably latter today)